btk/survey.py

```diff
@@ -0,0 +1,17 @@
+import descwl
+
+
+class Survey(descwl.survey.Survey):
```

Comment:
great! I think this is probably the simplest way without changing the code significantly